scst-store/using_encryption_and_connection.md

```diff
@@ -50,22 +50,32 @@ Where:
 
```

Comment:
In the CA Certificate section above this: there should be instruction on how to get the CA certificate if ingress in enabled.
ie:
kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > PATH

Reply:
I added that line